packages/cli/src/cli/github/github.ts

```diff
@@ -1,128 +1,73 @@
 import { Env, LogType } from '@basemaps/shared';
-import { Octokit } from '@octokit/core';
-import { Api } from '@octokit/plugin-rest-endpoint-methods/dist-types/types.js';
-import { restEndpointMethods } from '@octokit/plugin-rest-endpoint-methods';
-
-export const owner = 'linz'; // The Owner of the Github repository
-export const repo = 'basemaps-config'; // Github repository name
-export const base = 'master'; // Base head name of repository
-
-export interface Job {
-  imagery: string;
-  tileMatrix: string;
-  content: string;
-}
-
-export interface Blob {
-  path: string;
-  mode: '100644';
-  type: 'blob';
-  sha: string;
-}
+import { execFileSync } from 'child_process';
 
 export class Github {
-  octokit: Api;
+  repo = 'basemaps-config';
```

Comment:
we shouldnt hard code this, but if we are we should also include the org prefix. `linz/basemaps-config`